security-paseto/src/main/java/io/micronaut/security/token/paseto/config/PasetoConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ *  Licensed under the Apache License, Version 2.0 \(the "License"\);
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package io.micronaut.security.token.paseto.config;
+
+import io.micronaut.core.util.Toggleable;
+
+/**
+ * Represents configuration of the PASETO token.
+ *
+ * @author Utsav Varia
+ * @since 3.0
+ */
+public interface PasetoConfiguration extends Toggleable {
+
+    /**
+     *
+     * @return a boolean flag indicating whether PASETO beans should be enabled or not
+     */
+    @Override
+    boolean isEnabled();
```

Comment:
You can remove this method. The method is defined in the interface and we will set the Javadoc in the implementation.

Reply:
I should not have added to `JwtConfiguration` probably.